rdmo/core/assets/js/hooks/useLsState.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react'
+import { isEmpty, isPlainObject, omit as lodashOmit } from 'lodash'
+
+import { checkStoreId } from '../utils/store'
+
+const useLsState = (path, initialValue, omit = []) => {
```

Comment:
is this `localStorage` `useLsState` hook just a re-implementation of what was already stored in localStorage before or is it a new state altogether??

Reply:
Before we had `configActions` which are synced with the local storage. Here I bypass the redux store and just use a `useState` hook which is synced to the local storage.